FirebaseInAppMessaging/Sources/Public/FirebaseInAppMessaging/FIRInAppMessagingRendering.h

```diff
@@ -164,6 +164,13 @@ NS_SWIFT_NAME(InAppMessagingDisplayMessage)
 /// Unavailable.
 - (instancetype)init NS_UNAVAILABLE;
 
+/// Exposed for unit testing only. Don't instantiate this in your app directly.
+- (instancetype)initWithMessageID:(NSString *)messageID
```

Comment:
@ryanwilson Is this ok without an API review.

Would it be better to add it to a category inside the test instead of to the public header?

Reply:
Can't do it as a category unfortunately, since the tests are in Swift.

Reply:
Oh right, of course.

Reply:
Yeah, this needs an API review. Do you mind setting that up?

Reply:
Kicked off API review process, will track internally.